### DISCUSSION TITLE
Should we remove prettier?

### DISCUSSION THREAD

**Discussion by nordfjord** [equinox-project/equinox-js#71] [Polls]
Over time I've found that prettier produces more standardised code, but not necessarily more readable code.

Often this is due to the column width being fairly strictly adhered to. So adding an argument to a function can often lead to a fairly nasty line break

```diff
- const result = await this.writer.writeMessage(a, b, c)
+ const result = await this.writer.writeMessage(
+   a,
+   b,
+   c,
+   d,
+ )
```

As such I'm considering whether we should drop prettier entirely and rely more on discipline

**Comment by bartelink** [comment]
is there a more compact alias for prettier-ignore?
Either way I think prettier is great once you have contributors as it takes whitespace nits/style things out of the picture (and focuses any discussions on exceptions) - this is fine as long as you have 1/2 closely aligned people but becomes a slog otherwise

(Esp given how far you've got; I suspect that something reasonable might be attainable eventually on the F# side too - but to come up with something reasonable will require a bit more fantomas evolution, and some careful tuning, probably to an extent like  https://github.com/fsprojects/FSharp.Control.TaskSeq/blob/main/src/.editorconfig does)
